### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by Nicordr222** [Deadlineem/Extras-Addon-for-YimMenu#136] [General]
Is it normal that i cant see a few features from the features list? For example i cant see the selected player options.

**Comment by Deadlineem** [comment]
> Is it normal that i cant see a few features from the features list? For example i cant see the selected player options.

So the features list is way way waaaay outdated.  The selected player options should show up as colored tabs when you click a players name now compared to the original way the script was set up.

There are things like Player Options > fun self > magnet that will allow you to target the player and use them as well.

**Comment by Nicordr222** [comment]
Thank you have another question, and I’m not sure if this is possible or if it already exists. I’ve been looking for a way to save F1 wheels. At Benny’s, it works fine if you buy insurance after equipping them. However, while the open F1 wheels are saved, the 'Fukaru' or other branding prints do not carry over. Thanks in advance

**Reply by Deadlineem** [reply]
> Thank you have another question, and I’m not sure if this is possible or if it already exists. I’ve been looking for a way to save F1 wheels. At Benny’s, it works fine if you buy insurance after equipping them. However, while the open F1 wheels are saved, the 'Fukaru' or other branding prints do not carry over. Thanks in advance

I do not think so, this used to be a thing but since 1.69 it has been buggy, every time ive tried saving them they revert to stock after closing the game and reopening them, only the mods stay, wheels, plates and liveries haven't been.

**Reply by Nicordr222** [reply]
I just found a way if you use model changer change the car to a f1 car change wheels and them change it back to your old car hash and it stays after restart

**Reply by Deadlineem** [reply]
> I just found a way if you use model changer change the car to a f1 car change wheels and them change it back to your old car hash and it stays after restart

Should post a video tutorial on how to do it, alot of people would be interested in that fr.

**Reply by Nicordr222** [reply]
If you want i dont know if its possible to add a tool that makes it easier to swap hashes because you have to look for the right hashes on [rage](https://wiki.rage.mp/index.php?title=Vehicles)
